tools/accuracy_checker/accuracy_checker/preprocessor/audio_preprocessing.py

```diff
@@ -64,20 +64,76 @@ class ClipAudio(Preprocessor):
     def parameters(cls):
         parameters = super().parameters()
         parameters.update({
-            'duration': NumberField(
-                value_type=int, min_value=0,
-                description="Length of audio clip in seconds."
+            'duration': BaseField(
+                description="Length of audio clip in seconds or samples (with 'samples' suffix)."
             ),
             'max_clips': NumberField(
-                value_type=int, min_value=1, default=1,
+                value_type=int, min_value=1, optional=True,
                 description="Maximum number of clips per audiofile."
             ),
+            'overlap': BaseField(
+                optional=True,
+                description="Overlapping part for each clip."
+            ),
         })
         return parameters
 
     def configure(self):
-        self.duration = self.get_value_from_config('duration')
-        self.max_clips = self.get_value_from_config('max_clips')
+        duration = self.get_value_from_config('duration')
+        if isinstance(duration, str):
+            if duration.endswith('samples'):
+                try:
+                    self.duration_in_samples = int(duration[:-7])
+                except ValueError:
+                    raise ConfigError("Preprocessor {}: invalid value for duration - {}."
+                                      .format(self.__provider__, duration))
+                if self.duration_in_samples <= 1:
+                    raise ConfigError("Preprocessor {}: duration should be positive value - {}."
+                                      .format(self.__provider__, self.duration_in_samples))
+            else:
+                raise ConfigError("Preprocessor {}: invalid value for duration - {}.".
+                                  format(self.__provider__, duration))
+        else:
+            try:
+                self.duration = float(duration)
+            except ValueError:
+                raise ConfigError("Preprocessor {}: invalid value for duration - {}."
+                                  .format(self.__provider__, duration))
```

Comment:
probably duration configuration should be moved to independent function, too many logic it has to be a part of common configuration